string type  # Controllers => PID, PIS
---
bool success
string message

